odoo/addons/test_new_api/tests/test_many2many.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from odoo.tests.common import TransactionCase
+
+
+class many2manyCase(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.ship = self.env["test_new_api.ship"].create({'name': 'Colombus'})
+        self.env["test_new_api.pirate"].create({
+            'name': 'Barbe noire',
+            'ship_ids': self.ship,
+        })
+        self.env['test_new_api.prisonner'].create({
+            'name': 'Brian',
+            'ship_ids': self.ship,
+        })
+        self.pirate = self.env["test_new_api.pirate"].create({'name': 'Barbe rouge'})
+
+    def test_not_in_relation(self):
+        pirates = self.env["test_new_api.pirate"].search([('ship_ids', 'not in', self.ship.id)])
```

Comment:
`not in` expects a list of ids, give it a list of ids
```suggestion
        pirates = self.env["test_new_api.pirate"].search([('ship_ids', 'not in', self.ship.ids)])
```